src/Azos/Data/EntityId.cs

```diff
@@ -69,12 +74,12 @@ public EntityId(Atom sys, Atom type, string address)
                                                 Type.GetDistributedStableHash() ^
                                                 ShardKey.ForString(Address);
 
-    public override bool Equals(object obj) => obj is EntityId other ? this.Equals(other) : false;
+    public override bool Equals(object obj) => obj is EntityId other ? Equals(other) : false;
```

Comment:
I would keep `this.` as the code is harder to read, because there is "object.Equals" and "object.ReferenceEquals" methods, the line "this.Equals()" means specifically THIS instance OVERRIDE.

The practice is widely used in music notation, e.g. using "md/ms" designators denoting right and left hand application in keyboard arpeggios is not required but makes score MUCH easier to read at a first glance.

"this." is used in a similar fashion